ff_control/ff_control/tri_thruster_ctrl.py

```diff
@@ -0,0 +1,57 @@
+# MIT License
+#
+# Copyright (c) 2023 Stanford Autonomous Systems Lab
```

Comment:
Change license to 2024. And same for all other new files.

Reply:
For this license, do I only change the files that I worked on/changed? Or should I do this for anything my code interacts with?